packaging/installer/kickstart.sh

```diff
@@ -922,7 +924,7 @@ handle_existing_install() {
         INSTALL_PREFIX="${ndprefix}"
         claim
         ret=$?
-      elif [ "${NETDATA_CLAIM_ONLY}" -eq 1 ]; then
+      elif [ "${ACTION}" = "claim" ]; then
         fatal "User asked to claim, but did not proide a claiming token." F0202
```

Comment:
(Not of this PR but caught my eye): `proide` -> `provide`.

Reply:
@Ferroin do you plan to fix messages with this PR or another?

Reply:
I’ve got plans to go over all the messages in detail in another PR at this point.